user/enterprise/setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Setting up Travis CI Enterprise
+layout: en_enterprise
+redirect_from:
+  - /user/enterprise/installation/
+  - /user/enterprise/prerequisites/
+  - /user/enterprise/install-on-xenial/
+
+---
+
+Travis CI Enterprise works with your GitHub.com or GitHub Enterprise setup.
+
+## Prerequisites
+
+  * A valid Travis CI Enterprise license or [trial license](https://enterprise.travis-ci.com/signup). If you're interested in using [High Availability](/user/enterprise/high-availability/), [please let us know](mailto:enterprise@travis-ci.com) so we can get your trial license configured.
+
+  * At least two virtual machines on your private cloud that meet the [system requirements](#system-requirements)
+  * The `secret` and `clientid` of a GitHub.com or GitHub Enterprise [OAuth application](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/) configured with:
+    - *Homepage URL* - `https://travis-ci.<your-domain>.com`
+    - *Authorization callback URL* - `https://travis-ci.<your-domain>.com/api`
+
+### System Requirements
+
+The standard setup consists of the *Travis CI Enterprise
+Platform* which hosts the web UI and related services, and one or more
+*Worker hosts* which run the tests/jobs in isolated containers using LXC
+and Docker.
+
+Each dedicated host or hypervisor (VMWare, OpenStack using KVM, or EC2) should
+run **Ubuntu 14.04** or **Ubuntu 16.04**, ideally using Linux 3.16 and have at least **16 gigs of
```

Comment:
The machines should run 16.04 or later. It's time to decommission Trusty support soon.